common/scripts/base.sh

```diff
@@ -139,3 +138,25 @@ function showUsageAndExitDefault() {
   echoBold "Ex: ./deploy.sh"
   exit 1
 }
+
+function deploy_common_service()
+{
+  if ! bash ${mesos_artifacts_home}/common/${1}/deploy.sh; then
+    echoError "Non-zero exit code returned when deploying ${1}"
+    exit 1
+  fi
+}
+
+function deploy_wso2_service()
```

Comment:
why is it named "deploy_wso2_service" ? I don't think we need "wso2" part here. wdyt?